### DISCUSSION TITLE
How to access termux home from file browser?

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-app#3220] [Q&A] [answered]
Hi there.  Once again Thank you for termux.

Use it everyday.

I read that it's not possible to symlink termux home directory to emulated storage.

But is it possible to access termux $HOME directory from file browser?

I've tried solid explorer but it fails.

/data/data/com.termux/files/home/

Screenshot from solid explorer
https://www.dropbox.com/s/vxni55u9k3wj0o8/Screenshot_20230129_110910_Solid%20Explorer.jpg?dl=0

May be there is some workaround with accessing termux home not only from terminal and notepads.

**Comment by agnostic-apollo** [comment, marked as answer]
Check `Access Termux from a file manager`

https://wiki.termux.com/wiki/Internal_and_external_storage

**Comment by ghost** [comment]
Thank you!!!
Thanks to termux I've launched my find.dhamma.gift project on my phone and now can work completely offline. It's very cool for the forest environment. 
🙏🙏😊
--

BR
Pavel K
https://find.dhamma.gift - Liberation Search Engine. Поисковая Система Освобождения  Sunday, 29 January 2023, 01:51PM +07:00 from agnostic-apollo  ***@***.*** :

**Comment by agnostic-apollo** [comment]
Welcome.

**Comment by ghost** [comment]
By the way. Running website locally on termux is great but I guess average user will fail setting up everything and fixing errors.
Is it possible to create a snapshot or archive-like apk with everything installed and tuned already. So user will just install lets say termux-dhamma-gift and launch the app.

--

BR
Pavel K
https://find.dhamma.gift - Liberation Search Engine. Поисковая Система Освобождения  Tuesday, 31 January 2023, 02:58PM +07:00 from agnostic-apollo  ***@***.*** :

**Reply by agnostic-apollo** [reply]
Restoring `$PREFIX` could be done, but ideally setup scripts should be written.

https://wiki.termux.com/wiki/Backing_up_Termux
